tests/acceptance/parameters/test_quality_parameter.py

```
@@ -0,0 +1,63 @@
+        "entity": "106",
+        "name": "A space",
+        "entry_date": "2019-01-07",
+        "start_date": "2019-01-05",
+        "end_date": "2020-01-07",
+        "dataset": "greenspace",
+        "json": None,
+        "organisation_entity": None,
+        "prefix": "greenspace",
+        "reference": "Q1234567",
+        "typology": "geography",
+        "geometry": "MultiPolygon (((-0.3386878967285156 53.74426323597749, -0.337904691696167 53.743857158459996, -0.33673524856567383 53.744003093019586, -0.33637046813964844 53.74463124033804, -0.3365743160247803 53.74525937826645, -0.33737897872924805 53.74541799747043, -0.33875226974487305 53.74505000000031, -0.3386878967285156 53.74426323597749)))",  # noqa: E501
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)